main.go

```diff
@@ -8,7 +8,16 @@ import (
 
 func main() {
 	http.HandleFunc("/", func(writer http.ResponseWriter, request *http.Request) {
-		fmt.Fprintf(writer, "Hello %s", "chrons")
+		writer.Header().Add("Content-Type", "text/html")
+		html := `
```

Comment:
Can you serve HTML from public directory?
Thanks!

Reply:
Done.